pom.xml

```diff
@@ -54,9 +54,9 @@
         </contributor>
     </contributors>
     <properties>
-        <metrics.version>3.0.2</metrics.version>
+        <metrics.version>3.1.1</metrics.version>
         <jackson.version>2.2.2</jackson.version>
```

Comment:
Do we also want to bump jackson version to `2.4.2`? That's the current version for the dropwizard/metrics project. dropwizard is currently at `2.5.0`.